src/basic/cgroup-util.c

```diff
@@ -2211,6 +2212,62 @@ int cg_trim_everywhere(CGroupMask supported, const char *path, bool delete_root)
         return 0;
 }
 
+int cg_mask_to_string(CGroupMask mask, char **ret) {
+        _cleanup_strv_free_ char **t = NULL;
+        CGroupController c;
+        char *s;
+
+        assert(ret);
+
+        if (!mask)
+                return 0;
+
+        for (c = 0; c < _CGROUP_CONTROLLER_MAX; c++)
+                if (mask & CGROUP_CONTROLLER_TO_MASK(c)) {
```

Comment:
I'd always invert this check, so that we don#t have to indent so much... i.e. "if (!(mask & ...)) continue"...